src/CLU/Microsoft.CLU/CommandBinder/CmdletBinderAndCommand.cs

```diff
@@ -177,7 +177,7 @@ public IEnumerable<string> ListCommands(string[] args)
 #if PSCMDLET_HELP
             return CmdletHelp.Generate(parser.FormatParameterName, _discriminatorBinder.Modules, args, prefix);
 #else
-            var commands = Microsoft.CLU.Help.CommandDispatchHelper.CompleteCommands(CLUEnvironment.GetPackagesRootPath(), args).ToArray();
+            var commands = Microsoft.CLU.Help.CommandDispatchHelper.CompleteCommands(new HelpPackageFinder(CLUEnvironment.GetPackagesRootPath()), args).ToArray();
             if (commands.Length > 0)
```

Comment:
Factored out the package path into HelpPackageFinder so we can create our own packages in the unit tests